CONTRIBUTING.md

```diff
@@ -371,3 +370,14 @@ Also, we use [cargo-udeps](https://github.com/est31/cargo-udeps) to find unused
 workspace.
 
 We use [cargo-sort](https://crates.io/crates/cargo-sort) to ensure all deps are get sorted.
+
+## To check-in PRs from forks...
+
+```
+gh pr checkout <PR id>
+git checkout -b forks/<PR id>
```

Comment:
```suggestion
git switch -c forks/<PR id>
```

Reply:
I'd prefer to keep the working tree clean before doing the switch 🤪